.github/scripts/format-check.sh

```diff
@@ -4,6 +4,7 @@ unformatted() {
   set -x
   make format
   if [[ -n "$(git status --porcelain)" ]]; then
+    git --no-pager diff
```

Comment:
This will help with debugging when the code-formatting check fails, by showing the actual diff that CI is complaining about.